openapi3/validation_options.go

```diff
@@ -3,46 +3,57 @@ package openapi3
 import "context"
 
 // ValidationOption allows the modification of how the OpenAPI document is validated.
-type ValidationOption func(options *ValidationOptions)
+type ValidationOption func(options *validationOptions)
 
-// ValidationOptions provide configuration for validating OpenAPI documents.
-type ValidationOptions struct {
+type examplesValidation struct {
+	Disabled     bool
+	AsReq, AsRes bool
+}
+
+type validationOptions struct {
```

Comment:
Please keep that type exported. It's bad documentation practice to have public types that mention private types.